content/arduino-cloud/01.getting-started/09.arduino-cloud-for-business/contend.md

```diff
@@ -120,6 +120,8 @@ There you will find the complete list of all members who have access to your Sha
 
 There are multiple role types in Arduino Cloud for Business to allow the admin to have full control of the rights/permissions each user has in the Shared Space. In this way, the admin can set up Role-Based-Access-Control (RBAC) by assigning profiles and sharing with any number of users.
 
+By default the member who has activated the plan and created the workspace is set as Admin. The Role-Based-Access-Control can support multiple Admins with same permissions.
```

Comment:
I agree with this sentence, but, at this point, I'd suggest to remove the sentence after the table saying "Thus, the only role with full permissions is the *Admin*, which corresponds to the owner of the Arduino Cloud for Business plan." If you agree, @Fr4ng could you do that in the same branch please?